tracing/jaeger/logging.go

```diff
@@ -0,0 +1,21 @@
+package jaeger
+
+import (
+	"context"
+
+	"github.com/rs/zerolog"
+
+	"github.com/kiali/kiali/log"
+)
+
+// getLoggerFromContextGRPCJaeger returns the logger found in the context and adds the type key to refer to this as the gRPC Jaeger logger
+func getLoggerFromContextGRPCJaeger(ctx context.Context) *zerolog.Logger {
```

Comment:
nit that you don't have to change: these are a little verbose. Could just be a `getLogger` method on the client itself `func (jc JaegerHTTPClient) getLogger( ... ) ... `.